### PR TITLE
solc 0.5.9

### PR DESCRIPTION
**Pull request by rainbreak**

### CODE REVIEW

**Comment by rainbreak** [comment]
solc is already at 0.5.9 in nixos/nixpkgs master, so would I actually add directly to the `darwin` section?

**Comment by asymmetric** [comment]
Exactly, you would add it to `darwin` and point it to the commit on NixOS/nixpkgs instead of our fork.

**Comment by rainbreak** [comment]
would you like rebase on https://github.com/dapphub/dapptools/pull/230?

File: src/dapp/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.21.0] - 2019-06-21
+### Added
+- Support for solc 0.5.9
```
Comment (asymmetric) [inline src/dapp/CHANGELOG.md]: Can you add `Default to solc 0.5.9` to `### Changed`?

**Pull request merged**